stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/avatar/AvatarBitmapFactory.kt

```diff
@@ -355,6 +355,6 @@ public open class AvatarBitmapFactory(private val context: Context) {
         /**
          * Marker object to detect whether methods have been implemented by custom subclasses.
          */
-        private val NOT_IMPLEMENTED_MARKER = Bitmap.createBitmap(1, 1, Bitmap.Config.ALPHA_8)
+        private fun notImplementedMarker() = Bitmap.createBitmap(1, 1, Bitmap.Config.ALPHA_8)
```

Comment:
The point of stopping using `val` that doesn't have `const` inside companion objects, because garbage collector won't ever collect them. Each object takes very little memory, but its aggregation can make a difference in the end. 

It doesn't have to be a function, we can also move the variable inside the class instead of being an static reference

Reply:
Moved as class variable